packages/components/src/advanced-filters/attribute-filter.js

```diff
@@ -112,7 +112,7 @@ const AttributeFilter = ( props ) => {
 		}
 		setAttributeTerms( false );
 		apiFetch( {
-			path: `/wc-analytics/products/attributes/${ selectedAttribute[ 0 ].key }/terms`,
+			path: `/wc-analytics/products/attributes/${ selectedAttribute[ 0 ].key }/terms?per_page=-1`,
```

Comment:
```suggestion
			path: `/wc-analytics/products/attributes/${ selectedAttribute[ 0 ].key }/terms?per_page=100`,
```

Though, would it be better if we make it `100` so it would look more deliberate? `-1` does seem a bit misleading about the real limit.

Reply:
Sure thing :)